contracts/upgradeable_contracts/arbitrary_message/BasicHomeAMB.sol

```diff
@@ -37,6 +37,16 @@ contract BasicHomeAMB is BasicAMB, MessageDelivery {
         }
     }
 
+    /**
+    * @dev Requests message relay to the opposite network, message is sent to the manual lane.
+    * @param _contract executor address on the other side.
+    * @param _data calldata passed to the executor on the other side.
+    * @param _gas gas limit used on the other network for executing a message.
+    */
+    function requireToConfirmMessage(address _contract, bytes _data, uint256 _gas) external returns (bytes32) {
+        return requireToPassMessage(_contract, _data, _gas);
```

Comment:
does it make sense to add a comment here describing that the behavior of requireToPassMessage will be changed since internally it checks the selector of invoked method. But I would strongly recommend to change the code as so we manage the dataType byte more explicitly -- the current method is very smart but it could be difficult to keep such trick in the mind every time the code is read.

Reply:
Ok, I have changed the way it works, now it should be better.